### DISCUSSION TITLE
Keycloak 18.0.1 - administration dashboard fix

### DISCUSSION THREAD

**Discussion by jrivers96** [keycloak/keycloak#12527] [Archive]
Hi, 

Is there a due date for keycloak 18.0.1? I'm interested in a bug fix related to the administration dashboard. 

Thanks!

**Comment by ghost** [comment]
I'm also having an issue with the admin dashboard. Which issue is this?

**Comment by abstractj** [comment]
@jrivers96 could you please be more specific about which issue?

**Reply by ghost** [reply]
I'm having the same issue as the person who submitted issue #12733

**Comment by abstractj** [comment]
@cole-bermudez would you be able to try using `start-dev`?

```
FROM quay.io/keycloak/keycloak:18.0.2 as builder

ENV KC_HEALTH_ENABLED=true
ENV KC_METRICS_ENABLED=true
ENV KC_DB=postgres
# Install custom providers
RUN curl -sL https://github.com/aerogear/keycloak-metrics-spi/releases/download/2.5.3/keycloak-metrics-spi-2.5.3.jar -o /opt/keycloak/providers/keycloak-metrics-spi-2.5.3.jar
RUN /opt/keycloak/bin/kc.sh build

FROM quay.io/keycloak/keycloak:18.0.2
COPY --from=builder /opt/keycloak/ /opt/keycloak/
WORKDIR /opt/keycloak
ENTRYPOINT ["/opt/keycloak/bin/kc.sh", "start-dev"]
```

**Reply by ghost** [reply]
I actually got it fixed not long ago by using
KC_PROXY=edge
KEYCLOAK_HOSTNAME_ADMIN=keycloak.myadmindomain.com
KEYCLOAK_HOSTNAME_STRICT_HTTPS=false

as env variables in my docker-compose. I appreciate your help!

**Comment by abstractj** [comment]
@cole-bermudez can we close https://github.com/keycloak/keycloak/issues/12733 ?

**Reply by ghost** [reply]
I don't think that's my call to make.

**Comment by abstractj** [comment]
@cole-bermudez my apologies, I guess I misread your previous sentences.

**Comment by sylvaingirardbe** [comment]
I have the same issue as these people. Their fixes don't help me and running with `start-dev` doesn't give my any more info than running with `start`.

**Reply by sylvaingirardbe** [reply]
Ok, so I just got it running on `quarkus` by setting `KC_HOSTNAME` correctly instead of `localhost`.

**Comment by lucasscheepers** [comment]
I also have the same issue as these people. We're using [this helm chart](https://artifacthub.io/packages/helm/bitnami/keycloak) to deploy Keycloak on a Kubernetes platform. The helm chart we're using is version 9.3.6.

When I inspect the blank page and refresh the page I see it does an `http` call instead of `https` we're using.
